README.md

```diff
@@ -19,6 +19,11 @@
 
 ## Info
 
+*FAQ 
+
+"action controller doesn't recognize term input" 
```

Comment:
This isn't a question, but is located in the `FAQ` section. I'd recommend either moving it to a "troubleshooting" section or phrase it as a question.

Also, enclose the *actual command* in ticks. `db:drop db:create db:migrate` isn't a terminal program. You probably mean `rake db:drop db:create db:migrate` or something like that. Please update.